drivers/syslog/syslog_intbuffer.c

```diff
@@ -65,9 +65,10 @@ struct syslog_intbuffer_s
  * Private Data
  ****************************************************************************/
 
-static struct syslog_intbuffer_s g_si_buffer =
+static struct syslog_intbuffer_s g_syslog_intbuffer =
```

Comment:
but should we merge to the first patch?

Reply:
I don't want to merge. actually, I think the first commit is more perfect, and the second commit will have 2 meaningless line breaks.

![image](https://github.com/user-attachments/assets/bd9c2032-0d43-4d75-a3b3-434a63af1b0a)
![image](https://github.com/user-attachments/assets/c2717c2b-b769-416d-9d3f-ea06dfd8e6d3)